SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -449,19 +449,12 @@ void ActorSystem::HandleActorAuthority(const Worker_EntityId EntityId, const Wor
 						Actor->SetReplicates(true);
```

Comment:
I believe the lines above that reset `Role` and `RemoteRole` are now only necessary if load balancing is disabled. This was done previously to restore correct roles after they were flipped in `USpatialActorChannel::ReplicateActor`, but now that code looks like [this](https://github.com/spatialos/UnrealGDK/blob/master/SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp#L689-L695):
```
			// We preemptively set the Actor role to SimulatedProxy if load balancing is disabled
			// (since the legacy behaviour is to wait until Spatial tells us we have authority)
			if (NetDriver->LoadBalanceStrategy == nullptr)
			{
				Actor->Role = ROLE_SimulatedProxy;
				Actor->RemoteRole = ROLE_Authority;
			}
```
@m-samiec is it still necessary to support the case where the load balancing is disabled?
If not, the role flipping can be removed altogether on the auth server.